source/process/release-process.md

```diff
@@ -21,197 +21,178 @@ Pre-work for the current release begins at the code complete date of the previou
     - Draft release headline and send to marketing lead for review
-    - Queue Twitter banners in the Marketing: Graphics channel
+    - Decide which sections of the release announcement will have an accompanying screenshot / photo
 
 ### D. (T-minus 10 working days) Judgment Day
 
-Day when leads and PM area owners decide which major features are included in the release, and which are postponed.
+Day when Leads and PMs decide which major features are included in the release, and which are postponed.
 
-1. Release Manager:
-    - Post this checklist in Release channel
+1. **(Team) Judgment Day Meeting (10:00am San Francisco time)**:  
+    - Discuss worst bug on master
+    - Finalize which major features will be in or out for the release
+    - Begin daily triage of tickets
+2. Release Manager:
+    - Post this checklist in Release Checklist channel
     - Verify all items in the last posted release checklist are complete
-2. Logistics:
-    - Confirm, in Judgement Day meeting, date of marketing announcement for the release and update release channel header if needed
-    - Post an reminder to devs in the Release Discussion room of the the code complete date with the ZBB count
-3. Leads:
-    - Finalize roadmap for next release, and identify planned marketing bullet points  
-4. **(Team) Judgment Day Meeting (10:00am San Francisco time)**:  
-    - Meet to discuss release update and finalize which major features will be in or out for the release
-5. PM:
-    - Based on discussion, create tickets for features that need to be turned on or off for the release
-    - Backlog is reviewed and tickets that won’t make it are moved to next release
-    - Post a link to Release Discussion room for query of [remaining tickets in this release](https://mattermost.atlassian.net/issues/?filter=11102)
     - Update Changelog PR based on what's in/out of the release
     - Create meta issue for release in GitHub (see [example](https://github.com/mattermost/mattermost-server/issues/3702))
+3. Logistics:
+    - Confirm date of marketing announcement for the release date with Marketing, and update release channel header if needed
+    - Post a reminder to devs in the Release Discussion channel of the the code complete date with the ZBB count [see example](https://pre-release.mattermost.com/core/pl/coggyys9atg7fqyam81q3gkmoo)
+4. Leads:
+    - Finalize roadmap for next release, and identify planned marketing bullet points
+5. PM:
+    - Based on results of Team Meeting discussion, create tickets for features that need to be turned on or off for the release
     - Review the JIRA tickets remaining in the current release fix version and push those that won't make it to the next fix version
-    - Review any features that are currently in Beta and confirm with leads if there are any to be promoted
+    - Post a link to Release Discussion channel for query of [remaining tickets in this release](https://mattermost.atlassian.net/issues/?filter=11102)
```

Comment:
Is this useful to anyone?

Reply:
Or maybe we replace this item with posting the list of remaining bugs, which we do)